olm/build-bundles.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
```

Comment:
Some documentation such as

You need to have the following tools installed
* olm, see https://github.com/operator-framework/operator-lifecycle-manager/releases/latest

would be great, as well as a mention that you kube-context must point at an OS cluster

Reply:
Done.